lib/platform/bitbucket-server/index.js

```diff
@@ -651,6 +658,25 @@ async function createPr(
 ) {
   logger.debug(`createPr(${branchName}, title=${title})`);
   const base = useDefaultBranch ? config.defaultBranch : config.baseBranch;
+  let reviewers = [];
+
+  /* istanbul ignore else */
+  if (config.bbUseDefaultReviewers) {
+    logger.debug(`fetching default reviewer`);
+    const { id } = (await api.get(
```

Comment:
From this code I'm assuming that Bitbucket Server lets you configure a single default reviewer?

Reply:
nope, result is an array, you can configure multiple rules with multiple reviewers.

here we get all reviewers from matching rules.